pkg/agent/cniserver/pod_configuration.go

```diff
@@ -468,15 +468,15 @@ func (pc *podConfigurator) reconcile(pods []corev1.Pod, containerAccess *contain
 				continue
 			}
 			podWg.Add(1)
-			go func(containerID, pod, namespace string) {
+			go func(containerID, iface, pod, namespace string) {
```

Comment:
you probably can undo changes to this file if you use the correct interface store?